go/vt/vtgate/executor.go

```diff
@@ -63,6 +64,39 @@ import (
 // this is the healthcheck used by vtgate, used by the "vstream * from" functionality
 var vtgateHealthCheck discovery.HealthCheck
 
+var getTabletThrottlerStatus = func(tabletAddr string) (string, error) {
```

Comment:
I don't get it. Why isn't this just a normal function instead of being a function variable?

Reply:
Just noting that we discussed this via Slack and I will be making it a normal function. I wasn't sure why it was used here so just aped it originally: https://github.com/vitessio/vitess/blob/59031fd32f3e3333b1263b897bb9fc0bc8e7de74/go/vt/wrangler/version.go#L36

But given that function variables generally make sense when the function is ... variable (can be overridden/changed by user flags, actions, etc) it doesn't make sense to use it here. Thanks!